includes/general.php

```diff
@@ -2048,3 +2048,79 @@ function pods_session_start() {
 	return true;
 
 }
+
+/**
+ * Check if a field supports read or write via the REST API.
+ *
+ * @since 2.5.6
+ *
+ * @param string $field_name The field name.
+ * @param object|Pods $pod Pods object.
+ * @param bool|true $read Are we checking read or write?
+ *
+ * @return bool If supports, true, else false.
+ */
+function pods_rest_api_field_allowed_to_extend( $field_name, $pod, $read = true ) {
```

Comment:
maybe rename? it's about if a field can be accessed  for write and or read
i know true / false is more performant but it's harder to understand then 
e.g  $check = write | read  instead of $read = true | false  
but it's harder to read especially line 83/84 @PodsRESTFields